tutorials/io/ntuple/ntpl001_staff.C

```diff
@@ -33,7 +33,7 @@
 #include <utility>
 
 // Import classes from experimental namespace for the time being
-using RNTupleModel = ROOT::Experimental::RNTupleModel;
+using RNTupleModel = ROOT::RNTupleModel;
```

Comment:
I think we should qualify the classes directly in the code so it becomes copy-pasteable

Reply:
You resolved this thread, but I don't see the changes in the tutorials?